azkaban-web-server/src/main/less/header.less

```diff
@@ -67,5 +67,9 @@
     margin: 0px;
     padding: 0px;
     background: transparent;
+
+    .flowlink:before {
+      content: ":";
+    }
```

Comment:
To make sure all clients apply these style please update `v=...` to current timestamp in style.vm file.
`<link href="/css/azkaban.css?v=<current-timestamp>" rel="stylesheet">`

Reply:
Done.